### PR TITLE
feat:  Orders

### PR DESCRIPTION
**Pull request by Haider8**

### CODE REVIEW

**Comment by Haider8** [comment]
@iamareebjamal Please review

File: alembic/versions/3a0ebcec740d_orders_and_ordered_items_tables.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('order_datetime', sa.DateTime(), nullable=False),
+        sa.Column('user_id', sa.Integer(), nullable=False),
+        sa.Column('shop_id', sa.Integer(), nullable=False),
+        sa.Column('user_instructions', sa.String(length=150), nullable=True),
+        sa.Column('total_cost', sa.Numeric(scale=2), nullable=True),
+        sa.Column('accepted', sa.Boolean(), nullable=True),
+        sa.Column('declined', sa.Boolean(), nullable=True),
+        sa.Column('canceled', sa.Boolean(), nullable=True),
+        sa.Column('delivered', sa.Boolean(), nullable=True),
```
Comment (iamareebjamal) [inline alembic/versions/3a0ebcec740d_orders_and_ordered_items_tables.py]: This should be a single column, status

File: app/db_models/order.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    ordered_items = relation("OrderedItem", back_populates="order")
+    user_id = Column(Integer, ForeignKey('user.id'), nullable=False)
+    user = relation("User", back_populates="orders")
+    shop_id = Column(Integer, ForeignKey('shops.id'), nullable=False)
+    shop = relation("Shop", back_populates="orders")
+    user_instructions = Column(String(length=150))
+
+    @aggregated('ordered_items', Column(Numeric(asdecimal=True, scale=2)))
+    def total_cost(self):
+        return func.sum(OrderedItem.cost)
```
Comment (iamareebjamal) [inline app/db_models/order.py]: Shouldn't it be cost*quantity

Comment (Haider8) [reply app/db_models/order.py]: Indeed

File: alembic/versions/3a0ebcec740d_orders_and_ordered_items_tables.py (hunk 1 above)
Comment (Haider8) [reply alembic/versions/3a0ebcec740d_orders_and_ordered_items_tables.py]: Should I make it a choice type or an enum?

**Comment by Haider8** [comment]
@iamareebjamal please review

**Pull request merged**